src/test/java/org/isf/medicalsinventory/test/Tests.java

```diff
@@ -0,0 +1,236 @@
+		cleanH2InMemoryDb();
+	}
+	
+	@Test
+	public void testMedicalInventoryGets() throws Exception {
+		int code = setupTestMedicalInventory(false);
+		checkMedicalInventoryIntoDb(code);
+	}
+	
+	@Test
+	public void testIoGetMedicalInventories() throws Exception {
+		int id = setupTestMedicalInventory(false);
+		MedicalInventory foundMedicalinventory = medIvnIoOperationRepository.findById(id).orElse(null);
+		assertThat(foundMedicalinventory).isNotNull();
+		List<MedicalInventory> medicalInventories = medicalInventoryIoOperation.getMedicalInventory();
+		assertThat(medicalInventories.get(medicalInventories.size() - 1).getId()).isEqualTo((Integer) id);
+	}
+	
+	@Test
+	public void testIoNewMedicalInventory() throws Exception {
+		Ward ward = testWard.setup(false);
+		MedicalInventory medicalInventory = testMedicalInventory.setup(ward, false);
+		MedicalInventory newMedicalInventory = medicalInventoryIoOperation.newMedicalInventory(medicalInventory);
+		checkMedicalInventoryIntoDb(newMedicalInventory.getId());
+	}
+	
+	@Test
+	public void testIoUpdateMedicalInventory() throws Exception {
+		Integer id = setupTestMedicalInventory(false);
+		MedicalInventory foundMedicalInventory = medIvnIoOperationRepository.findById(id).orElse(null);
+		assertThat(foundMedicalInventory).isNotNull();
+		String status = "canceled";
+		foundMedicalInventory.setStatus(status);
+		MedicalInventory updatedMedicalInventory = medicalInventoryIoOperation.updateMedicalInventoryRow(foundMedicalInventory);
+		assertThat(updatedMedicalInventory.getStatus()).isEqualTo(status);
+	}
+	
+	@Test
+	public void testIoDeleteMedicalInventory() throws Exception {
+		Integer id = setupTestMedicalInventory(false);
+		MedicalInventory foundMedicalInventory = medIvnIoOperationRepository.findById(id).orElse(null);
+		assertThat(foundMedicalInventory).isNotNull();
+		String reference = foundMedicalInventory.getInventoryReference();
+		medicalInventoryIoOperation.deleteMedicalInventory(foundMedicalInventory);
+		assertThat(medicalInventoryIoOperation.referenceExists(reference)).isFalse();
+	}
+	
+	@Test
+	public void testIoGetMedicalInventoryWithStatus() throws Exception {
+		int id = setupTestMedicalInventory(false);
+		MedicalInventory firstMedicalInventory = medIvnIoOperationRepository.findById(id).orElse(null);
+		assertThat(firstMedicalInventory).isNotNull();
+		Ward ward = testWard.setup(false);
+		MedicalInventory inventory = testMedicalInventory.setup(ward, false);
+		String status = "validated";
+		inventory.setStatus(status);
+		MedicalInventory secondMedicalInventory = medIvnIoOperationRepository.saveAndFlush(inventory);
+		assertThat(secondMedicalInventory).isNotNull();
+		List<MedicalInventory> medicalInventories = medicalInventoryIoOperation
+				.getMedicalInventoryByStatus(firstMedicalInventory.getStatus());
+		assertThat(medicalInventories.get(0).getStatus()).isEqualTo(firstMedicalInventory.getStatus());
+		assertThat(medicalInventories.size()).isEqualTo(1);
```

Comment:
```suggestion
		assertThat(medicalInventories).hasSize(1);
```
When testing the size of `List` objects it is better to use `hasSize()`.